applications/luci-app-vnstat2/htdocs/luci-static/resources/view/vnstat2/config.js

```diff
@@ -0,0 +1,113 @@
+'require tools.widgets as widgets';
+
+return L.view.extend({
+	handleDeleteModal: function(m, iface, ev) {
+		L.showModal(_('Delete interface <em>%h</em>').format(iface), [
+			E('p', 'The interface will be removed from the database permanently. This cannot be undone.'),
+			E('div', { 'class': 'right' }, [
+				E('div', {
+					'class': 'btn',
+					'click': L.hideModal
+				}, _('Cancel')),
+				' ',
+				E('div', {
+					'class': 'btn cbi-button-negative',
+					'click': ui.createHandlerFn(this, 'handleDelete', m, iface)
+				}, _('Delete'))
+			])
+		]);
+	},
+
+	handleDelete: function(m, iface, ev) {
+		return fs.exec('/usr/bin/vnstat', ['--remove', '-i', iface, '--force'])
+			.then(L.bind(m.render, m))
+			.then(L.hideModal)
+			.catch(function(e) {
+				L.hideModal();
+				ui.addNotification(null, E('p', e.message));
+			});
+	},
+
+	render: function() {
+		var m, s, o;
+
+		m = new form.Map('vnstat', _('vnStat'), _('vnStat is a network traffic monitor for Linux that keeps a log of network traffic for the selected interface(s).'));
+
+		s = m.section(form.TypedSection, 'vnstat', _('Interfaces'));
+		s.anonymous = true;
+		s.addremove = false;
+
+		o = s.option(widgets.DeviceSelect, 'interface', _('Monitor interfaces'), _('The selected interfaces are automatically added to the vnStat database upon startup.'));
+		o.rmempty = true;
+		o.multiple = true;
+		o.noaliases = true;
+		o.nobridges = false;
+		o.noinactive = false;
+		o.nocreate = false;
+
+
+		o = s.option(form.DummyValue, '_database');
+
+		o.load = function(section_id) {
+			return Promise.all([
+				fs.exec('/usr/bin/vnstat', ['--json', 'f', '1']),
+				uci.load('vnstat')
+			]).then(L.bind(function(results) {
+				var databaseInterfaces = [];
+				if (results[0].code == 0) {
+					var vnstatData = JSON.parse(results[0].stdout);
+					for (var i = 0; i < vnstatData.interfaces.length; i++) {
+						databaseInterfaces.push(vnstatData.interfaces[i].name);
+					}
+				}
+
+				var configInterfaces = uci.get_first('vnstat', 'vnstat', 'interface') || [];
+
+				this.interfaces = databaseInterfaces.filter(function(iface) {
+					return configInterfaces.indexOf(iface) == -1;
+				});
+			}, this));
+		};
+
+		o.render = L.bind(function(view, section_id) {
+			var table = E('div', { 'class': 'table' }, [
+				E('div', { 'class': 'tr table-titles' }, [
+					E('div', { 'class': 'th' }, _('Interface')),
+					E('div', { 'class': 'th right' }, _('Delete'))
+				])
+			]);
+
+			var rows = [];
+
+			for (var i = 0; i < this.interfaces.length; i++) {
+				rows.push([
+					this.interfaces[i],
+					E('button', {
+						'class': 'btn cbi-button-remove',
+						'click': ui.createHandlerFn(view, 'handleDeleteModal', m, this.interfaces[i])
+					}, [ _('Delete…') ])
+				]);
+			}
+
+			cbi_update_table(table, rows, E('em', _('No unconfigured interfaces found in database.')));
+
+			return E([], [
+				E('h3', _('Unconfigured interfaces')),
+				E('div', { class: 'cbi-section-descr' },
```

Comment:
Can you put the `class:` field in single quotes? Iirc an unquoted "class" causes problems in some environment since it is a keyword in later ECMAScript versions.